django_filters/rest_framework/backends.py

```diff
@@ -49,8 +50,10 @@ def filter_queryset(self, request, queryset, view):
         filter_class = self.get_filter_class(view, queryset)
 
         if filter_class:
-            return filter_class(request.query_params, queryset=queryset, request=request).qs
-
+            filterset = filter_class(request.query_params, queryset=queryset, request=request)
+            if not filterset.is_valid() and self.raise_exception:
+                raise utils.translate_validation(filterset.errors)
+            return filterset.qs
```

Comment:
Also I noticed that even if I do not want to raise exception here, then it is not raised, but queryset returned without filtering. I guess in case if invalid filterset and do not raise exception it should return `qs.none()`

Reply:
It keeps the partial queryset, yet - I think this is tested for in https://github.com/carltongibson/django-filter/pull/788/files#diff-b18add7202661e9eacece0aded8e619bR301.

Reply:
Agree with @sshishov , if `strict=True` shouldn't be returning `queryset.none()` here?